tests/Old/CommonExtractorTest.php

```diff
@@ -32,6 +32,18 @@ public function setUp(): void
         $this->initDatabase();
     }
 
+    public function tearDown(): void
```

Comment:
We usually do our cleaning up in `setUp` rather than `tearDown` so that if a test fails or errors, the following test is guaranteed to still have a clean run-space.  But maybe this is a topic for #best-practices

Reply:
definitely setup